colossalai/booster/plugin/moe_hybrid_parallel_plugin.py

```diff
@@ -68,8 +67,36 @@ def __init__(
         self.pp_pg = pp_process_group
         if use_pipeline:
             init_pipeline_optimizer(optimizer, model)
+
+        zero_params = list(filter(lambda x: not is_moe_tensor(x), model.parameters()))
```

Comment:
perhaps for now we can throw runtime error here in case that user have more than one torch.Optimizer.param_group?

Reply:
done.